BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -345,6 +345,19 @@ - (NSString *)fullItemFieldsParameterString
     return [[set array] componentsJoinedByString:@","];
 }
 
+- (NSString *)fullItemFieldsParameterStringExcludingFields:(NSArray *)excludedFields
+{
+    NSMutableOrderedSet *set = [NSMutableOrderedSet orderedSet];
+    [set addObjectsFromArray:[self fullFolderFieldsArray]];
+    NSArray * fileFields = [self fullFileFieldsArray];
+    if (excludedFields.count > 0) {
+        fileFields = [[NSMutableArray arrayWithArray:allFileFields] removeObjectsInArray:excludedFields];
```

Comment:
right now we only apply this to file. Should we build the combined set, then remove excluded fields from the union then turn it into an array?